docs/tracked-extension-resources-are-not-allowed.md

```diff
@@ -0,0 +1,25 @@
+# trackedExtensionResourcesAreNotAllowed
+
+## Category
+
+ARM Error
+
+## Applies to
+
+ARM OpenAPI (swagger) specs
+
+## Related ARM Guideline Code
+
+- RPC-Uri-V1-12
+
+## Output Message
+
+{0} is an extension resource and the response schema in {1} operation includes location property. Extension resources of type tracked are not allowed.
+
+## Description
+
+Extension resources cannot be of type tracked.
+
+## How to fix the violation
+
+Remove "location" property from the property bag of an extension resource.
```

Comment:
>property bag [](http://example.com/codeflow?start=36&length=12)

location is a top level property and not in the property bag. change it to

Remove the top level "location" property from the model of the extension resource.